types/koa-ratelimit/koa-ratelimit-tests.ts

```diff
@@ -18,6 +18,13 @@ app.use(rateLimit({
     db: new Map()
 }));
 
+app.use(rateLimit({
+    driver: 'memory',
+    db: new Map(),
+    blacklist: (context) => Promise.resolve(true),
+    whitelist: (context) => Promise.resolve(false)
+}));
```

Comment:
I think the tests should include both synchronous and asynchronous implementations. Something like:

```
blacklist: async (ctx) => true,
whitelist: (ctx) => false
```